src/main/java/mate/academy/bookshop/dto/CreateBookRequestDto.java

```diff
@@ -1,13 +1,19 @@
 package mate.academy.bookshop.dto;
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 public class CreateBookRequestDto {
+    @NotNull
     private String title;
+    @NotNull
     private String author;
     private String isbn;
```

Comment:
@NotNull(message = "ISBN cannot be null.")
    @Size(min = 1, max = 13, message = "ISBN must be between 1 and 13 characters.")